src/components/heading/heading.scss

```diff
@@ -66,3 +66,50 @@
   text-align: left;
   width: 50px;
 }
+
+// When inside a dialog full screen
+.carbon-dialog-full-screen__content {
+  > .carbon-carousel {
+    .carbon-heading {
+      padding-bottom: 0;
+    }
+  }
+
+  .carbon-heading {
+    height: 98px;
+
+    .carbon-heading__header {
+      background-color: $grey-light;
+      display: flex;
+      align-items: center;
+      height: 120px;
+      padding-left: 5%;
+      padding-right: 5%;
+      position: absolute;
+      left: 0;
+      top: 0;
+      width: calc(100% - 10%);
+
+      @media screen and (-ms-high-contrast: active), screen and (-ms-high-contrast: none) {
+        /* IE10+ specific styles go here */
```

Comment:
but the media query would apply to IE10 also so I thought the comment was relevant